### DISCUSSION TITLE
Simple B&W Landscape Theme

### DISCUSSION THREAD

**Discussion by MAHAON26** [mathoudebine/turing-smart-screen-python#595] [Themes]
I made simple black and white 8 titles theme. In "Backgrounds" folder you can find .psd to add custom photo to your background. Can be simply be edited using Photopea online. 

Theme was inspired by @Jojo10011 and his "Simple Dark Theme" (Thx❤)

Special thx to @elschnitzel for help with stats stuttering - 💕

![preview](https://github.com/user-attachments/assets/6610b9ff-0c68-47a8-833b-8cba7dd3f956)

For some reason Fps metre don't work on my PC, don't mater which theme I use, so I will be very happy to hear that its working on other setups (If you know solution for this problem - pls write it in comments)

Waiting for your reactions and possible improvement suggestions) 
 
Download:
[Simple B&W Landscape Theme.zip](https://github.com/user-attachments/files/17600466/Simple.B.W.Landscape.Theme.zip)

**Comment by DJRoby19** [comment]
Great job, FPS doesn't work for me on Linux either

**Comment by Deny1006** [comment]
Any chanse you could upload version with date and time instead of FPS? I was trying but failed

**Comment by sevagjb** [comment]
thanks for this nice theme, my GPU not working on the laptop
i have GTX 360
any advice to fix it?
